safe_transaction_service/history/services/collectibles_service.py

```diff
@@ -442,7 +504,7 @@ def get_collectibles_with_metadata_paginated(
         )
 
     @cachedmethod(cache=operator.attrgetter("cache_token_info"))
-    @cache_memoize(60 * 60, prefix="collectibles-get_token_info")  # 1 hour
+    @cache_memoize(TOKEN_EXPIRATION, prefix="collectibles-get_token_info")  # 1 hour
```

Comment:
In the code above token_expiration was 30 minutes here 1 hour, what is the correct value? @Uxio0